common/include/nbl/examples/geometry/CSimpleDebugRenderer.hpp

```diff
@@ -97,14 +97,26 @@ class CSimpleDebugRenderer final : public core::IReferenceCounted
 			// load shader
 			smart_refctd_ptr<IShader> shader;
 			{
-				const auto bundle = assMan->getAsset("nbl/examples/geometry/shaders/unified.hlsl",{});
-// TODO: Arek
-				//const auto bundle = assMan->getAsset("nbl/examples/geometry/shaders/unified.spv",{});
+				// TODO & NOTE: tmp, maybe I will turn it into CMake option
+				#define NBL_USE_PRECOMPILED_SPIRV
+
+				#ifdef NBL_USE_PRECOMPILED_SPIRV
+				constexpr std::string_view key = "nbl/examples/shaders/geometry/unified.hlsl.spv";
+				#else
+				constexpr std::string_view key = "nbl/examples/shaders/geometry/unified.hlsl";
+				#endif // NBL_USE_PRECOMPILED_SPIRV
```

Comment:
We have nothing to gain from NOT compiling SPIR-V  if given the chance.

Example should know if it wants SPIR-V or HLSL (runtime compilation, but from embedded source)

I just need the option to do both HLSL (src or include archive) and SPIR-V (build dir archive) as I please, and I hardcode on case-by-case basis